### ISSUE TITLE
Replace usage of `select` with an accessible dropdown on the home feed

### ISSUE THREAD

**Issue by aitchiss** [forem/forem#13605]
There are a few places in the app which show a `select` for navigation when we're in a mobile screen size, in place of a set of tabs or other UI. As pointed out on #11853, `select` probably isn't the best thing to use in this situation because it:

- suggests a form with some kind of submit mechanism
- stops links from being links (as they're surfaced as 'options' instead)

This issue relates to the Home feed, but I will raise similar for other places where we use this pattern.

We should replace the select on the home feed (that's used to select between Feed/Week/Month/etc) with an accessible crayons dropdown (guidance on how to implement accessibly is in our storybook). 

NB: this will result in a visual change in the UI if we run with the crayons dropdown component's normal styling. Pinging @forem/product for confirmation on whether this is OK.

**Comment by lisasy** [comment]
@aitchiss If you're open to it, I want to close this issue because we will be solving this by deprecating this dropdown component completely. May I close this issue?

https://share.getcloudapp.com/Jru401pb

**Comment by aitchiss** [comment]
@lisasy Yep, definitely! Consider it closed - looking forward to all the good things to come 😄
